src/DrawManager.sol

```diff
@@ -209,28 +218,51 @@ contract DrawManager {
     uint24 drawId = prizePool.getDrawIdToAward(); 
     uint48 closesAt = prizePool.drawClosesAt(drawId);
```

Comment:
```suggestion
    uint48 drawClosesAt = prizePool.drawClosesAt(drawId);
```

This should be clarified to separate it from the `closedAt` param in the StartDrawAuction struct.